apps/omg_watcher/test/omg_watcher/integration/block_getter_test.exs

```diff
@@ -79,7 +79,7 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
              %{"blknum" => ^block_nr}
            ] = TestHelper.get_utxos(alice.addr)
 
-    assert TestHelper.get_utxos(alice.addr) == TestHelper.get_exitable_utxos(alice.addr)
+    assert Enum.sort(TestHelper.get_utxos(alice.addr)) == Enum.sort(TestHelper.get_exitable_utxos(alice.addr))
```

Comment:
let's Enum.sort in the endpoint (the `get_exitable_utxos` one, which is which became unsorted in RocksDB). I think `get_utxos` returns them sorted and so should `get_exitable_utxos`

Reply:
I've done this on rocksdb after iteration. It happens because of the way we append results at the end of the list!